### ISSUE TITLE
grpc4bmi error running docker

### ISSUE THREAD

**Issue by jeromaerts** [eWaterCycle/grpc4bmi#44]
The following problem occurs when running the ewatercycle/pcrg-grpc4bmi:latest docker:

Traceback (most recent call last):
  File "/usr/local/bin/run-bmi-server", line 9, in <module>
    load_entry_point('grpc4bmi==0.2', 'console_scripts', 'run-bmi-server')()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2569, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/grpc4bmi/run_server.py", line 21, in <module>
    from .bmi_grpc_server import BmiServer
  File "/usr/local/lib/python2.7/dist-packages/grpc4bmi/bmi_grpc_server.py", line 293
    def __repr__(self) -> str:
                       ^
SyntaxError: invalid syntax

**Comment by sverhoeven** [comment]
Some Python 3 type annotations slipped in.

**Comment by sverhoeven** [comment]
Working on PR to fix it. 

At the moment the master branch of this repo uses the newest BMI spec, see #41, which requires changes to the model and Dockerfile. 

To use an older version of grpc4bmi replace line https://github.com/eWaterCycle/grpc4bmi-examples/blob/master/pcrglob/Dockerfile#L6
with 
```
RUN pip install git+https://github.com/eWaterCycle/grpc4bmi.git@0.2#egg=grpc4bmi
```

**Comment by sverhoeven** [comment]
The Syntax error has been fixed in master branch with #45 PR
